evalml/utils/gen_utils.py

```diff
@@ -270,3 +270,27 @@ def is_all_numeric(df):
         if dtype not in numeric_and_boolean_dtypes:
             return False
     return True
+
+
+def _convert_nullable_types_wrapper(pd_data):
```

Comment:
Yupperino! I updated `test_targets_data_types_classification` to include these types.

(I wanted to keep the testing of target types and data types in one place rather than creating a new test, but it is kinda growing out of hand...)